package.json

```
@@ -39,7 +39,7 @@
   "dependencies": {
     "@eclipse-che/workspace-client": "^0.0.1-1608729566",
     "@patternfly/patternfly": "^4.10.31",
-    "@patternfly/react-core": "^4.18.5",
+    "@patternfly/react-core": "^4.23.1",
```

Actually, resolved version is `4.84.4`, so let it be put in product.json along with `~` to allow updates for patch releases. 

```suggestion
    "@patternfly/react-core": "~4.84.4",
```